config/stage.json

```diff
@@ -92,7 +92,7 @@
 		"layout/query-selected-editor": true,
 		"layout/support-article-dialog": true,
 		"layout/dotcom-nav-redesign-v2": false,
-		"layout/wpcom-admin-interface": false,
+		"layout/wpcom-admin-interface": true,
```

Comment:
I think we're turning on the wrong flag? 😂

Reply:
🤦‍♂️